lib/Gym/space_handler.ex

```diff
@@ -0,0 +1,28 @@
+defmodule Gyx.Gym.Utils do
+  @moduledoc """
+  This module contains auxiliary functions to achieve full
+  compatibility with Gym, including functions to obtain
+  Gyx space representations from Gym space specs.
+  """
+  @space_types [{~r/Discrete\((?<n>\d+)\)/, :discrete}]
+
+  @doc """
+  This function takes the __repr__ response from Gym spaces
+  and creates an equivalent Gyx Space struct
+  """
+  def gyx_space(gym_space_string) do
+    gym_space_string |> parse |> create_space
+  end
+
+  def parse(gym_space_string) do
+    {regex, type} =
+      Enum.find(@space_types, {nil, :unknown}, fn {regex, _} ->
+        Regex.match?(regex, Kernel.inspect(gym_space_string))
+      end)
+
+    {type, Regex.named_captures(regex, Kernel.inspect(gym_space_string))}
+  end
+
+  defp create_space({:discrete, capture}), do: %Gyx.Core.Spaces.Discrete{n: capture["n"]}
```

Comment:
Esto ya no se si sea buena práctica, o idea mía, pero por lo general en Elixir, pienso que es preferible obtener el elemento de un mapa por medio de Map|Dict|Keyword.get en lugar de el azúcar sintáctica. El motivo es que parece más funcional, y me ayuda a saber con que estructura de datos estoy lidiando, pero en realidad no tengo una opinión fuerte en eso..

Reply:
Agree, done

Reply:
I meant using Keyword.get(capture, "n") instead of `capture["n"]`.. but this is the only elixir code I have seen recently..